simtools/util/names.py

```diff
@@ -748,3 +755,44 @@ def camera_efficiency_log_file_name(site, telescope_model_name, zenith_angle, la
     name += f"_{label}" if label is not None else ""
     name += ".log"
     return name
+
+
+def get_telescope_type(telescope_name):
+    """
+    Guess telescope type from name. Types are "LST", "MST", "SST", "SCT".
+    Parameters
+    ----------
+    telescope_name: str
+        Telescope name
+    Returns
+    -------
+    str
+        Telescope type.
+    """
+
+    _class, _ = split_telescope_model_name(telescope_name)
+    try:
+        if _class[0:3] in ("LST", "MST", "SST", "SCT"):
+            return _class[0:3]
+
+    except IndexError:
+        pass
+
+    return ""
+
+
+def get_corsika_telescope_data_dict():
+    io_handler = simtools.io_handler.IOHandler()
+    io_handler.set_paths(output_path="./simtools-output/", data_path="./data/")
+    file_data = collect_data_from_yaml_or_dict(
+        io_handler.get_input_data_file("parameters", "corsika_parameters.yml"), None
+    )
+    keys = ["corsika_sphere_center", "corsika_sphere_radius"]
+    final_dict = {}
+    for key in keys:
+        final_dict[key] = {}
+        for tel_type in file_data[key].keys():
+            final_dict[key][tel_type] = file_data[key][tel_type]["value"] * u.meter
+    final_dict["corsika_obs_level"] = file_data["SITE_PARAMETERS"]["North"]["OBSLEV"][0] * u.cm
+    final_dict["corsika_obs_level"] = final_dict["corsika_obs_level"].to(u.m) * u.meter
```

Comment:
Not sure about this - isn't it already in cm?

Reply:
it is in cm but the astropy.units is not read automatically. but I saw an error here and corrected it: I did not need to multiply again "* u.meter" since it is already in meter (given the '.to(u.m) '). thanks for pointing this out